pages/platform/ai/deploy_guide_05_app_portfolio/guide.de-de.md

```diff
@@ -25,6 +25,24 @@ For example, you can rely on **open-source** models or apps.
 
 ## Portfolio of AI apps and models
 
+To test **AI Deploy**, you can quickly deploy apps based on those proposed in our portfolio.
+
+### Quick examples
+
+| Owner | Task | Description | Documentation | Dockerfile | Docker image | HTTP port | CLI command | Usage |       
+|-------|------|-------------|---------------|------------|--------------|-----------|-------------|-------|
+| OVHcloud | Hello world | Launch your first API with Flask | https://docs.ovh.com/de/publiccloud/ai/deploy/getting-started/ | https://github.com/ovh/ai-training-examples/blob/main/apps/flask/hello-world/Dockerfile | `priv-registry.gra.training.ai.cloud.ovh.net/ai-deploy-portfolio/ai-deploy-hello-world` | 5000 | `ovhai app run --default-http-port 5000 priv-registry.gra.training.ai.cloud.ovh.net/ai-deploy-portfolio/ai-deploy-hello-world` | **API** - interact with the API with a curl command or a Python script |
```

Comment:
```suggestion
| OVHcloud | Hello world | Launch your first API with Flask | [AI Deploy - Getting started](https://docs.ovh.com/de/publiccloud/ai/deploy/getting-started/) | <https://github.com/ovh/ai-training-examples/blob/main/apps/flask/hello-world/Dockerfile> | `priv-registry.gra.training.ai.cloud.ovh.net/ai-deploy-portfolio/ai-deploy-hello-world` | 5000 | `ovhai app run --default-http-port 5000 priv-registry.gra.training.ai.cloud.ovh.net/ai-deploy-portfolio/ai-deploy-hello-world` | **API** - interact with the API with a curl command or a Python script |
```
Please update all the docs.ovh.com links so they are clickable and contain an alt text.
GitHub links should be clickable too.